### DISCUSSION TITLE
[#1166491] Two (possibly) Border Block bugs.

### DISCUSSION THREAD

**Discussion by G01dY05h1** [Mojang/minecraft-editor#80] [Bugs]
1. (Most likely a bug) While in editor mode, the invisible border above and below the Border Block stops you from selecting things beyond it as if there were a block there.

2. (Least likely a bug) While in Crosshair mode, a player who is not the Operator cannot pass through the border. This is intended. The (possible) problem is that a non-Operator player can simply fly though it like any other block. I assume this one is not a bug, but I thought I should mention it regardless.

**Comment by minecraftchloe** [comment]
Recorded in our system!

**Comment by minecraftchloe** [comment]
We dug into this! 

1. This is as designed and we made some updates to selection logic, so it _should_ be working as designed now, where you're able to select things beyond. 
2. We compared with the main/retail game (non-Editor) with Editor. tl;dr Editor test mode should reflect main/retail gameplay, but anything in Editor Tool or Crosshair mode will not discriminate member vs operator. 

In main/retail game: 
- In_ Operator mode a player can move above and below border_blocks
- In Member mode a player cannot move above or below border_blocks
- And both Operator and Member a player can physically see the blocks and the floating debris

Editor: 
- In Crosshair mode when in Editor, as an Operator and Member, a player **can** move above and below border_blocks - this is expected because it's in Editor mode and we aren't discriminating by role during Editing/building. 
- Only in Editor when you are in test mode, can a player in member mode not be able to move above or below border blocks. Test mode in Editor should reflect gameplay in the main/retail game.
